src/apis/utils/date.utils.ts

```diff
@@ -0,0 +1,11 @@
+import moment from "moment";
+
+
+export const  DateFormate = (date:string)=>{
+    
```

Comment:
Remove empty lines & as it is single line return function skip return keyword & { } brackets aswell
export const DateFormat = (date: string) => date ? moment(date).format('lll') : "_";